README.md

```diff
@@ -1,2 +1,243 @@
-# math-magicians-project
+<a name="readme-top"></a>
+
+<div align="center">
+ <br>
+ <h1 style="color:rgb(87, 247, 255);border: 3px solid rgb(87, 247, 255);">Math Magicians</h1>
+ <br>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Math Magicians (Math-Magicians)](#Math-Magicians)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#live-demo)
+  - [🔭 Walk through video](#walk-through-video)
+  - [💻 Getting Started ](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+ 
+<!-- PROJECT DESCRIPTION -->
+<br>
+
+# 📖 Math Magicians (Math-Magicians) <a name="Math-Magicians"></a>
+
  "Math Magicians" is an interactive React website designed for mathematics enthusiasts. It serves as a Single Page App (SPA) where users can perform basic calculations and explore random quotes related to mathematics.
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The following stacks were used
+
+<details style="color:rgb(87, 247, 255);">
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+    <li><a href="hhttps://create-react-app.dev/">Create React App</a></li>
+ </ul>
+</details>
+
+<!-- Features -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **ReactJS library**
+- **Using JSX syntax**
+- **Unit test with jest**
+- **Boostrap**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+<br>
+
+# 💻 Getting Started <a name="getting-started"></a>
+
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```bash
+  cd my-folder
+  git clone git@github.com:PabloBona/math-magicians.git
+```
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** To make your documentation looks professional and descriptive, kindly ensure that you correct the typo error in your cloning link. Your project is named `Math-Magicians` and not `math-magicians`. So, the should be updated to this one `git clone git@github.com:PabloBona/Math-Magicians.git`. Also remember to add a command that allows use to access the cloned directory before starting install dependencies i.e. `cd Math-Magicians`(you can put it at the `line 94`)

Reply:
Thanks, Tresor, I do the changes now!